### PR TITLE
FIX: correctly supports dropping image

### PR DESCRIPTION
**Pull request by jjaffeux**
`UppyImageUploader` used to support this but this has been broken.

This commit:
- Adds back the target drop options to enable the feature
- Applies the css to every elements and not just the one for admin emojis, also fixes the style as it was flashing and preventing it to work. For now we just change the color of the image icon.
- Adds a test to ensure we don't regress.

### CODE REVIEW

File: app/assets/javascripts/discourse/app/components/uppy-image-uploader.js [hunk 1]
```diff
@@ -35,6 +35,9 @@ export default class UppyImageUploader extends Component {
       type: this.type,
       additionalParams: this.additionalParams,
       validateUploadedFilesOptions: { imagesOnly: true },
+      uploadDropTargetOptions: () => ({
+        target: document.querySelector(".uploaded-image-preview"),
```
Comment (Drenmi) [inline app/assets/javascripts/discourse/app/components/uppy-image-uploader.js:39]: Would there be a problem using a class name if there are multiple uppys on the same page?

**Review by Drenmi — APPROVED** [review]
LGTM! Just one hypothetical question.

File: app/assets/javascripts/discourse/app/components/uppy-image-uploader.js (hunk 1 above)
Comment (jjaffeux) [reply app/assets/javascripts/discourse/app/components/uppy-image-uploader.js:39]: hmmm yes that's a good questions 🤔 need to check, and think of how to fix it if that's a problem :+1:

**Review by Drenmi — APPROVED** [review]
🚀

**Pull request merged**